freshmaker/pyxis.py

```diff
@@ -205,9 +207,16 @@ def get_manifest_list_digest_by_nvr(self, nvr, must_be_published=True):
             for repo in image['repositories']:
                 if must_be_published and not repo['published']:
                     continue
-                if 'manifest_list_digest' in repo:
-                    return repo['manifest_list_digest']
-        return None
+                if 'manifest_list_digest' not in repo:
+                    continue
+
+                mld = repo['manifest_list_digest']
+                msd = None
+                if 'manifest_schema2_digest' in repo:
+                    msd = repo['manifest_schema2_digest']
+
+                return mld, msd
+        return None, None
```

Comment:
And what if `manifest_list_digest` is present in the image and `manifest_schema2_digest` isn't and vice versa? We still should return (None None)? As for now, I see that the code(`get_bundles_by_digest`) can work with just `manifest_list_digest` and without schema2.